Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10-alpine
```

Comment:
Do we need node alpine? We might be able to use another image (something smaller, if it exists), although it probably doesn't really matter :)

Reply:
node docker image is used because later in deploy script it is installing the surge package.

Reply:
It maybe doesn't matter here, but Alpine is one of the most problematic distributions for Python packages, because it's musl rather than glibc based so `manylinux` wheels don't work if they contain compiled code. Just FYI

Reply:
Thanks for the info.